src/pages/Map.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { GoogleMap, useLoadScript } from '@react-google-maps/api';
+import { mapStyles } from '../styling/mapStyles';
+
+const containerStyle = {
+  width: '100vw',
+  height: 'calc(100vh - 50.4px)',
+  position: 'absolute',
+  top: '50.4px'
+};
```

Comment:
can we avoid hardcoding this somehow? why is the height like that

Reply:
If we did something like change to fixed - would it make the other pages look better? If so, I'd vote for reformatting